app/legacy_lib/insert_payout.rb

```diff
@@ -75,6 +75,8 @@ def self.with_stripe(np_id, data, options={})
           bank_name: data[:bank_name]}])
         .returning('id', 'net_amount', 'nonprofit_id', 'created_at', 'updated_at', 'status', 'fee_total', 'gross_amount', 'email', 'count', 'stripe_transfer_id', 'user_ip', 'ach_fee', 'bank_name')
       ).first
+      # Create 'payout.created' ObjectEvent
+      Payout.publish_created
```

Comment:
I think this should be after the `pps` line but I don't think it matters that much.

Reply:
Also, it should be specific to a specific payout. We're doing a publish_created event for for a specific payout, not for all payouts.